offchainreporting2/oracle.go

```diff
@@ -39,6 +39,9 @@ type OracleArgs struct {
 	// Logger logs stuff.
 	Logger commontypes.Logger
 
+	// Enables adding metrics to track. This may be nil.
+	Metrics commontypes.Metrics
```

Comment:
Where is this used? Based on the comment about nil, I would expected to see a nil check somewhere in this PR, but can't find one. What am I missing?